# This contains the actual status of the gps

Header header

string fix_quality
string fix3d
string status
int32 sat_num
float32 latitude
float32 longitude
int32 utmzone
float32 X
float32 Y
float32 speed
float32 orientation